readme.md

```diff
@@ -99,6 +99,14 @@ Properties from `options` will override properties in the parsed `url`.
 
 If no protocol is specified, it will default to `https`.
 
+##### endpoint
+
+Type: `string` `Object`
+
+When specified, `url` will be preceded by `endpoint`.
+
```

Comment:
Can you also mention that this one is especially useful with `got.extend()` to create niche specific Got instances?